decidim-conferences/app/controllers/decidim/conferences/admin/conference_user_roles_controller.rb

```diff
@@ -46,7 +46,7 @@ def update
           enforce_permission_to :update, :conference_user_role, user_role: @user_role
           @form = form(ConferenceUserRoleForm).from_params(params)
 
-          UpdateConferenceAdmin.call(@form, @user_role) do
+          UpdateConferenceAdmin.call(@form, @user_role, current_conference) do
```

Comment:
As commented before, `current_conference` can be removed:
```suggestion
          UpdateConferenceAdmin.call(@form, @user_role) do
```